checkbox/snap/snapcraft.yaml

```diff
@@ -0,0 +1,65 @@
+name: checkbox-npu
+summary: Checkbox tests for the intel-npu-driver snap
+description: |
+  Collection of tests using Intel's user mode driver and compiler validation tool
+version: '1.0.0'
+confinement: classic
+grade: stable
+
+base: core22
```

Comment:
Change to checkbox24 if agreed that it should get bumped. Though since you have this all working, maybe it would be good to just let this land and then bump it in a later PR. This could potentially be nontrivial

Reply:
Done